services/movie.py

```
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids is None and actors_ids is None:
+        return queryset
+
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

The `get_movie_by_id` function uses `Movie.objects.get(id=movie_id)`, which will raise a `DoesNotExist` exception if the movie ID is not found. Consider using `get_object_or_404` if this function is used in a view, or handle the exception appropriately to avoid server errors.